File: LENTABP10/src/main/java/com/lenta/lentabp10/MobileObjectsService/models/MB_S_35.java
package com.lenta.lentabp10.MobileObjectsService.models;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;

// Модель справочника настроек заданий на списание WOB (справочник ГИС-контроль)
public class MB_S_35 {
    /// Тип задания на списание
    @JsonProperty("TASK_TYPE")
    @Getter private String taskType;

    /// Тип контроля задания на списание
    @JsonProperty("TASK_CNTRL")
    @Getter private String taskCntrl;

    public MB_S_35(String taskType, String taskCntrl) {
        this.taskType = taskType;
        this.taskCntrl = taskCntrl;
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/mobileObjectsService/models/MB_S_35.kt
package com.lenta.bp10.mobileObjectsService.models

import com.google.gson.annotations.SerializedName

class MB_S_35(
        /// Тип задания на списание
        @SerializedName("TASK_TYPE")
        val taskType: String,

        /// Тип контроля задания на списание
        @SerializedName("TASK_CNTRL")
        val taskCntrl: String
)